recipes/pymonctl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "PyMonCtl" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Kalmat/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
There is a wheel in your source distribution: https://github.com/Kalmat/PyMonCtl/tree/v0.7/dist

That makes downloading it a bit heavier and installation can be contaminated but it. Can you remove it?

PS: I did not check the others but my guess is that they also contain wheels in the sdist.

Reply:
Yes i noticed this too, but they should not end up in the final conda package? I am not the maintainer of the upstream repositories, so i can't remove them from the repository.

Reply:
Can you report it?

Reply:
https://github.com/Kalmat/PyWinCtl/issues/81